src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -8,6 +8,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <NoWarn>1591</NoWarn>
     <CLSCompliant>false</CLSCompliant>
+    <IsTargetingNetCoreApp Condition="'$(TargetsNetCoreApp)'=='true' OR '$(TargetsUap)' == 'true'">true</IsTargetingNetCoreApp>
```

Comment:
Nit: I would call this `TargetsNetCoreAppOrUap`. Bonus points for fixing up System.Numeric.Vectors.csproj to match.